model/symflower/symflower_test.go

```diff
@@ -257,3 +260,38 @@ func TestCountCharactersOfGeneratedFiles(t *testing.T) {
 		ExpectedCount: 169,
 	})
 }
+
+func TestSymflowerUnitTestsWithTimeout(t *testing.T) {
```

Comment:
Not a big fan of having some functionality and then rebuilding the same functionality for a tests.